test/release/examples/primers/fileIO.chpl

```diff
@@ -153,7 +154,7 @@ if example == 0 || example == 2 {
 // Now close the file, even though they are reference-counted like channels.
 // We can also remove the test file.
   f.close();
-  unlink(testfile);
+  FileSystem.remove(testfile);
```

Comment:
I would expect, if you were going to write it this way, you would import FileSystem above. But since you use'd it, I'd expect this to be just `remove(testfile)`.

Reply:
I saw that in most other primers we seemed to use "use", and that ``modules.chpl`` was the only one that ``import``ed.

If I update this to just say ``remove(testfile)`` I would also be inclined to update the comment. So something like this:
```chpl
// We can also remove the test file with ``FileSystem.remove``
```

I could also update that to be ``:proc:`FileSystem.remove` ``.

Reply:
Yes updating the comment sounds good to me. The link sounds nice too.